kong/plugins/datadog/schema.lua

```diff
@@ -79,6 +79,9 @@ return {
           { service_name_tag = { type = "string", default = "name" }, },
           { status_tag = { type = "string", default = "status" }, },
           { consumer_tag = { type = "string", default = "consumer" }, },
+          { retry_count = { type = "integer", default = 10 }, },
```

Comment:
We need to update `kong/clustering/compat/removed_fields.lua`.

Reply:
removed_fields added.